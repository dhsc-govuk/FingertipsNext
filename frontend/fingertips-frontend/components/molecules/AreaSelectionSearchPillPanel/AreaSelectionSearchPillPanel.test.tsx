import { render } from '@testing-library/react';
import { AreaSelectionSearchPillPanel } from './index';

describe('Test AreaSelectionSearchPillPanel', () => {
  it('take a snapshot of component and it renders correctly', () => {
    const container = render(
      <AreaSelectionSearchPillPanel
        areas={[
          {
            areaCode: 'GPs',
            areaName: 'Some gp practice here',
            areaType: 'some types information',
          },
        ]}
        onRemovePill={jest.fn()}
      />
    );
    expect(container.asFragment()).toMatchSnapshot();
  });

  it('test that can pill can be deleted when the close button is clicked', () => {});
});
