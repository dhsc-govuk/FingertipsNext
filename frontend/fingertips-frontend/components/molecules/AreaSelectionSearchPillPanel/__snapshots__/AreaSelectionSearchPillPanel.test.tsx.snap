// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AreaSelectionSearchPillPanel take a snapshot of component and it renders correctly 1`] = `
<DocumentFragment>
  .c6 {
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 16px;
  line-height: 1.25;
  margin-bottom: 5px;
}

.c3 {
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 16px;
  line-height: 1.25;
  color: #0b0c0c;
  margin-top: 0;
  margin-bottom: 15px;
  padding-left: 20px;
}

.c2 .c2 {
  margin-top: 10px;
}

.c3 > .c5 {
  margin-bottom: 5px;
}

.c12 {
  margin: 0;
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 16px;
  line-height: 1.25;
  margin-bottom: 15px;
}

.c12 > p {
  margin: 0;
}

.c12 > p > code {
  padding: 0.2em 0.4em;
  margin: 0;
  font-size: 85%;
  background-color: rgba(27,31,35,0.05);
  border-radius: 3px;
}

.c12 > pre {
  padding: 16px;
  overflow: auto;
  font-size: 85%;
  line-height: 1.45;
  background-color: #f6f8fa;
  border-radius: 3px;
}

.c12 > pre > code {
  display: inline;
  padding: 0;
  margin: 0;
  border: 0;
  overflow: visible;
  line-height: inherit;
  word-wrap: normal;
}

.c0 {
  padding: 0px;
  font-size: 19px;
}

.c10 {
  cursor: pointer;
}

.c8 {
  background-color: white;
  border: 1px #D1D2D3 solid;
  border-radius: 5px;
  padding: 0.3125em 0.3125em;
  max-width: 100%;
  margin: 0.3125em 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c11 {
  word-wrap: break-word;
  padding-left: 0.5em;
}

.c9 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c13 {
  margin-bottom: 0;
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 14px;
  line-height: 1.1428571428571428;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column-reserve;
  -ms-flex-direction: column-reserve;
  flex-direction: column-reserve;
  padding: 0px;
  margin: 0px;
  over-flow: hidden;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  margin-top: 10px;
  margin-bottom: 5px;
}

@media print {
  .c6 {
    font-size: 14px;
    line-height: 1.15;
  }
}

@media only screen and (min-width:641px) {
  .c6 {
    font-size: 19px;
    line-height: 1.3157894736842106;
  }
}

@media print {
  .c3 {
    font-size: 14px;
    line-height: 1.15;
  }
}

@media only screen and (min-width:641px) {
  .c3 {
    font-size: 19px;
    line-height: 1.3157894736842106;
  }
}

@media print {
  .c3 {
    color: #000;
  }
}

@media only screen and (min-width:641px) {
  .c3 {
    margin-bottom: 20px;
  }
}

@media print {
  .c12 {
    font-size: 14px;
    line-height: 1.15;
  }
}

@media only screen and (min-width:641px) {
  .c12 {
    font-size: 19px;
    line-height: 1.3157894736842106;
  }
}

@media only screen and (min-width:641px) {
  .c12 {
    margin-bottom: 20px;
  }
}

@media print {
  .c13 {
    font-size: 14px;
    line-height: 1.2;
  }
}

@media only screen and (min-width:641px) {
  .c13 {
    font-size: 16px;
    line-height: 1.25;
  }
}

<div>
    <h3
      class="c0 c1"
    >
      Selected areas (
      <span
        style="margin: 0px;"
      >
        1
      </span>
      )
    </h3>
    <ul
      class="c2 c3 c4"
    >
       
      <li
        class="c5 c6 c7"
      >
        <div
          class="c8"
          data-testid="pill-container"
        >
          <div
            class="c9"
            data-testid="remove-icon-div"
          >
            <svg
              class="c10"
              data-testid="x-icon"
              fill="none"
              height="12"
              stroke="#000000"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="3"
              viewBox="0 0 25 25"
              width="12"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="18"
                x2="6"
                y1="6"
                y2="18"
              />
              <line
                x1="6"
                x2="18"
                y1="6"
                y2="18"
              />
            </svg>
          </div>
          <div
            class="c11"
            data-testid="filter-name"
          >
            <div
              class="c12 c13"
            >
              <p>
                Some gp practice here-GPs
              </p>
            </div>
          </div>
        </div>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
